src/annotator/config/settings.js

```diff
@@ -10,10 +10,10 @@ function settingsFrom(window_) {
   var configFuncSettings = configFuncSettingsFrom(window_);
 
   /**
-   * Return the href URL of the first annotator link in the given document.
+   * Return the href URL of the first annotator sidebar link in the given document.
```

Comment:
Yeah I am going to leave the type piece alone for consistency sake. Migrating that on the extension and client can be done at a later time. I will go ahead and update the new rel to be `rel="hypothesis-client"`.